client/src/featureform/serving.py

```diff
@@ -99,7 +99,7 @@ def _local_training_set(self, trainingSetName, trainingSetVariant):
                 df.set_index(feature_row[9])
 
                 df.rename(columns={feature_row[11]: name_variant}, inplace=True)
-                feature_df = df#.sort_values(by=feature_row[10], ascending=False)
+                feature_df = df
```

Comment:
Is there still a reason for this?

Reply:
Just for clarity mainly, but unnecessary